ast/compile_test.go

```diff
@@ -1769,7 +1784,11 @@ func TestCompilerCheckSafetyBodyErrors(t *testing.T) {
 			sort.Strings(expected)
 
 			// Compile test module.
-			popts := ParserOptions{AllFutureKeywords: true, unreleasedKeywords: true}
+			popts := ParserOptions{
+				RegoVersion:        RegoV0,
```

Comment:
Is there a reason we specify the rego version here instead of updating the module to import `rego.v1`?

Reply:
I don't recall why I locked this down to v0. Doesn't need to be. I've corrected this.